Dockerfile.mongo

```
@@ -0,0 +1,24 @@
+FROM mongo:latest
+
+RUN apt-get update && apt-get install -y \
+    gnupg \
+    curl \
+    sudo \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN curl -fsSL https://www.mongodb.org/static/pgp/server-8.0.asc | \
```

Hadolint (2.12.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)